mono/metadata/appdomain.c

```diff
@@ -44,7 +44,6 @@
 #include <mono/metadata/threads.h>
 #include <mono/metadata/threadpool.h>
 #include <mono/metadata/tabledefs.h>
-#include <mono/metadata/gc-internals.h>
 #include <mono/metadata/mono-gc.h>
```

Comment:
True, and Flint does have an “Include What You Use” mode. However, I only removed includes when there was an identical, unconditional include in the same file.